demo_run.py

```diff
@@ -0,0 +1,277 @@
+    -------
+    The `lsst.verify.Job` object contained in `fileName`.
+    """
+    with open(fileName) as handle:
+        return Job.deserialize(**json.load(handle))
+
+
+def merge(jobs, lastJob):
+    """Combine measurements from multiple chips or visits.
+
+    Other job properties will be dictionary-merged (i.e., if multiple entries
+    are assigned to the same key, only one will be preserved).
+
+    Parameters
+    ----------
+    jobs: iterable of `lsst.verify.Job`
+        The jobs containing data to combine.
+    lastJob:
+        The job corresponding to the final run of ap_verify.
+
+    Return
+    ------
+    A single `lsst.verify.Job` object containing merged measurements from
+    `jobs`.
+    """
+    merged = Job.load_metrics_package()
+    # Visible Job state:
+    #     job.measurements
+    #     job.meta
+    #     job.metrics (guaranteed by load_metrics_package)
+    #     job.specs (guaranteed by load_metrics_package)
+
+    measurementsPerMetric = defaultdict(list)
+    for job in jobs:
+        for metricName in job.measurements:
+            measurementsPerMetric[str(metricName)].append(job.measurements[metricName])
+
+    for metric in measurementsPerMetric:
+        # Running times, object counts
+        if metric.endswith("Time") or metric in {
+                "ip_diffim.numSciSources",
+                "association.numNewDiaObjects",
+                "association.totalUnassociatedDiaObjects"}:
+            addIfDefined(merged.measurements,
+                         sumMeasurements(measurementsPerMetric[metric]))
+
+    # Fractions require special handling
+    addIfDefined(
+        merged.measurements,
+        # Due to time constraints, no metric for total DIAObjects was implemented,
+        # so we have to work around its absence
+        mergeFractionsPartial(
+            measurementsPerMetric["association.fracUpdatedDiaObjects"],
+            measurementsPerMetric["association.numUnassociatedDiaObjects"]))
+    addIfDefined(
+        merged.measurements,
+        mergeFractions(
+            measurementsPerMetric["ip_diffim.fracDiaSourcesToSciSources"],
+            measurementsPerMetric["ip_diffim.numSciSources"]))
+
+    # L1 database metrics are cumulative, not per-CCD, so just copy them over
+    for metric in ["association.totalUnassociatedDiaObjects"]:
+        if metric in lastJob.measurements:
+            addIfDefined(merged.measurements, lastJob.measurements[metric])
+
+    for job in jobs:
+        merged.meta.update(job.meta)
+
+    return merged
+
+
+def addIfDefined(measurementSet, measurement):
+    """Adds a measurement to a set if it is not None, else does nothing.
+    """
+    if measurement is not None:
+        measurementSet.insert(measurement)
+
+
+def sumMeasurements(measurements):
+    """Adds the values of some measurements.
+
+    Extras and notes will be dictionary-merged (i.e., if multiple extras or
+    notes are assigned to the same key, only one will be preserved).
```

Comment:
which one?  the last one?

Reply:
I'm not confident in guaranteeing that, especially since, if any extras or notes differ, this behavior is undesirable no matter which one gets chosen.